src/App.js

```diff
@@ -6,6 +6,10 @@ import Login from "./Pages/Login/Login";
 import SignUp from "./Pages/SignUp/SignUp";
 import Mockman from 'mockman-js';
 import Archive from "./Pages/archivePage/archive";
+import Trash from "./Pages/Trash/trash";
+import Label from "./Pages/Label/Label";
+import Account from "./Pages/Account/Account";
+import Sidebar from "./components/Sidebar/Sidebar";
```

Comment:
suggestion: use index file to export all pages from it in pages folder

Reply:
ok, I will work on that.